google-cloud-dialogflow/lib/google/cloud/dialogflow/v2/agent_pb.rb

```diff
@@ -5,6 +5,7 @@
 require 'google/protobuf'
 
 require 'google/api/annotations_pb'
+require 'google/api/resource_pb'
```

Comment:
This is a new annotation port, and we will need to set a minimum dependency on the common ports gem to ensure that this generated file exists to be required.